packages/EasyAsync/src/Bridge/Laravel/Providers/EasyAsyncServiceProvider.php

```diff
@@ -44,9 +44,9 @@ public function register(): void
             DateTimeGeneratorInterface::class => DateTimeGenerator::class,
             JobFactoryInterface::class => JobFactory::class,
             JobLogFactoryInterface::class => JobLogFactory::class,
-            UuidGeneratorInterface::class => RamseyUuidGenerator::class,
             'default_job_log_updater' => JobLogUpdater::class,
             JobLogUpdaterInterface::class => WithEventsJobLogUpdater::class,
+            RandomGeneratorInterface::class => RandomGenerator::class,
```

Comment:
@DKeeper We don't need to do that in this package as it is done in the `eonx-com/easy-random` package.

Reply:
@natepage If I remove this line, an error will be thrown when testing the package.
```
vendor/bin/phpunit packages/EasyAsync
...
1) EonX\EasyAsync\Tests\Bridge\Laravel\Providers\EasyAsyncServiceProviderTest::testServiceProvider with data set "Doctrine" ('doctrine', array('EonX\EasyAsync\Implementation...leaner', 'EonX\EasyAsync\Implementation...sister', 'EonX\EasyAsync\Implementation...sister'), Closure Object (...))
Illuminate\Contracts\Container\BindingResolutionException: Target [EonX\EasyRandom\Interfaces\RandomGeneratorInterface] is not instantiable while building [EonX\EasyAsync\Implementations\Doctrine\DBAL\DataCleaner, EonX\EasyAsync\Implementations\Doctrine\DBAL\JobLogPersister].
```

Reply:
@DKeeper yes and it makes sense because right now you have nothing to register this service in your application during your test.

How do you usually register 3rd party services into your Laravel/Lumen application?